docs/docs/connected-accounts/README.md

```diff
@@ -4,73 +4,69 @@ Pipedream allows you to connect accounts for various apps and services within ou
 
```

Comment:
Screenshots and references in this doc were out of date, so I fixed those and added a few useful sections.